fictadvisor-api/src/v2/api/controllers/DisciplineTeacherController.ts

```diff
@@ -307,4 +309,59 @@ export class DisciplineTeacherController {
     const updatedComment = await this.disciplineTeacherService.updateComment(disciplineTeacherId, body);
     return this.questionMapper.getUpdatedComment(updatedComment);
   }
+
+  @ApiBearerAuth()
+  @ApiOkResponse()
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException:
+      User with such id is not found
+      Question with such id is not found
+      DisciplineTeacher with such id is not found
+      
+    InvalidTypeException
+      Question has wrong type`,
+  })
+  @ApiForbiddenResponse({
+    description: `
+      NoPermissionException: You do not have permission to perform this action
+    `,
+  })
+  @ApiUnauthorizedResponse({
+    description: `\n
+    UnauthorizedException:
+      Unauthorized`,
+  })
+  @ApiForbiddenResponse({
+    description: `\n
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
+  @ApiParam({
+    name: 'questionId',
+    required: true,
+    description: 'Question Id',
+  })
+  @ApiParam({
+    name: 'disciplineTeacherId',
+    required: true,
+    description: 'Discipline teacher id',
+  })
+  @ApiParam({
+    name: 'userId',
+    required: true,
+    description: 'User id',
+  })
+  @ApiEndpoint({
+    summary: 'Delete question answers with TEXT type (comments)',
+    permissions: PERMISSION.COMMENTS_DELETE,
+  })
+  @Delete('/:questionId/:disciplineTeacherId/:userId/comment')
+  deleteComment(
+    @Param('questionId', QuestionByIdPipe) questionId: string,
+    @Param('disciplineTeacherId', DisciplineTeacherByIdPipe) disciplineTeacherId: string,
+    @Param('userId', UserByIdPipe) userId: string
+  ) {
```

Comment:
Need to add body object with information about comment (questionId, etc) for deletion